00_Installation/README.md

```diff
@@ -42,12 +42,17 @@ If you do not have git installed you will need to [install git]().  Once you hav
 
 These packages are reqorded in the requirements files in this directory, and can be installed using pip.
 
-You will need python version **3.8.10**. 
 We recommend the following command sequence, which creates a conda environment and installs the requirements into it.  
 
 ```
-% conda create -n aas239-jwebbinar python=3.8.10 pip wheel numpy
```

Comment:
Won't this cause python to default to 3.10? Last I saw, there were still missing wheels for 3.10, which caused build errors

Reply:
Hmm, when I did it just now it defaulted to 3.9 .  But it's a good point that might change between now and the day of the workshop, so maybe ``python=3.9`` is the way to go?

Reply:
Commit I pushed up does this, and also adds a bit of context to hopefully help